cmd/tailscale/cli/up.go

```diff
@@ -45,6 +45,7 @@ specify any flags, options are reset to their default.
 		upf.BoolVar(&upArgs.acceptRoutes, "accept-routes", false, "accept routes advertised by other Tailscale nodes")
 		upf.BoolVar(&upArgs.acceptDNS, "accept-dns", true, "accept DNS configuration from the admin panel")
 		upf.BoolVar(&upArgs.singleRoutes, "host-routes", true, "install host routes to other Tailscale nodes")
+		upf.StringVar(&upArgs.exitNode, "exit-node", "", "Tailscale IP of the exit node for internet traffic")
```

Comment:
is this an interim requirement or will we also be able to use a machine name (MagicDNS base name) here in the future? I made "tailscale ping" accept either recently (even if MagicDNS is off or --accept-dns=false)

Reply:
I'm a bit wary of using names, because those are mutable and so it brings up questions like "did you mean the hostname called Foo at the time you ran the command, or the one currently called Foo?" IPs are nice because they're immutable after node creation.

Not opposed to making names work too in a followup though.